src/components/commerce/CommerceSchedulesList.js

```diff
@@ -88,12 +88,9 @@ class CommerceSchedulesList extends Component {
         // de vigencia es pasada, al modificarlo en realidad se crea uno nuevo cuya fecha de inicio
```

Comment:
Aca de la 85 a la 95 seria:

`let startDate = selectedSchedule.startDate;

if (startDate < formattedMoment()) {
    startDate = formattedMoment();
}

this.props.onScheduleValueChange({ ...selectedSchedule, startDate });`